jjb/dynamic/scale-ci_install_gcp.yml

```diff
@@ -57,10 +57,6 @@
         default: true
         description: Creates workload and infra nodes post install.
         name: OPENSHIFT_POST_INSTALL
-    - bool:
```

Comment:
Maybe just remove it from our baremetal stuff @mukrishn I'm not sure if @chaitanyaenr and team are using OPENSHIFT_POST_CONFIG, maybe not?

Reply:
okay I will revert it, i thought you added it everywhere.